apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageMetaDataDownloader.java

```diff
@@ -0,0 +1,195 @@
+                                NBS_ODSE.dbo.WA_NND_metadata.hl7_segment_field, NBS_ODSE.dbo.WA_NND_metadata.order_group_id, NBS_ODSE.dbo.WA_UI_metadata.admin_comment,
+                                NBS_SRTE.dbo.Codeset.value_set_code,
+                                NBS_SRTE.dbo.Codeset.value_set_nm,
+                                NBS_ODSE.dbo.WA_UI_metadata.repeats_ind_cd,
+                                NBS_ODSE.dbo.WA_UI_metadata.part_type_cd,
+                                WA_UI_metadata.standard_question_ind_cd,
+                                NBS_ODSE.dbo.WA_UI_metadata.standard_nnd_ind_cd,
+                                NBS_ODSE.dbo.WA_UI_metadata.coinfection_ind_cd,
+                                NBS_ODSE.dbo.WA_NND_metadata.repeat_group_seq_nbr,
+                                NBS_ODSE.dbo.WA_UI_metadata.question_group_seq_nbr,
+                                NBS_ODSE.dbo.WA_UI_metadata.batch_table_appear_ind_cd,
+                                NBS_ODSE.dbo.WA_UI_metadata.batch_table_column_width,
+                                NBS_ODSE.dbo.WA_UI_metadata.batch_table_header,
+                                NBS_ODSE.dbo.WA_NND_metadata.question_map,
+                                NBS_ODSE.dbo.WA_UI_metadata.desc_txt,
+                                NBS_ODSE.dbo.WA_UI_metadata.question_type,
+                                NBS_ODSE.dbo.WA_NND_metadata.indicator_cd,
+                                NBS_ODSE.dbo.WA_UI_metadata.nbs_ui_component_uid,
+                                NBS_ODSE.dbo.WA_UI_metadata.code_set_group_id,
+                                NBS_ODSE.dbo.WA_UI_metadata.version_ctrl_nbr,
+                                NBS_ODSE.dbo.WA_UI_metadata.legacy_data_location,
+                                NBS_ODSE.dbo.WA_UI_metadata.entry_method,
+                                NBS_ODSE.dbo.WA_UI_metadata.question_oid,
+                                NBS_ODSE.dbo.WA_UI_metadata.question_oid_system_txt
+                                 ORDER BY participation_type, NBS_ODSE.dbo.WA_UI_metadata.order_nbr
+            """;
+
+
+    public ByteArrayInputStream downloadPageMetadataByWaTemplateUid(Long waTemplateUid) throws IOException {
+        final CSVFormat format = CSVFormat.Builder.create().setQuoteMode(QuoteMode.MINIMAL).
+                setHeader(PAGE_METADATA_HEADER.split(",")).build();
+        try (ByteArrayOutputStream out = new ByteArrayOutputStream();
+             CSVPrinter csvPrinter = new CSVPrinter(new PrintWriter(out), format)) {
+            List<Object[]> pageMetadata = findPageMetadataByWaTemplateUid(waTemplateUid);
+            for (Object[] data : pageMetadata) {
+                csvPrinter.printRecord(data);
+            }
+            csvPrinter.flush();
+            return new ByteArrayInputStream(out.toByteArray());
+        } catch (IOException e) {
+            throw new IOException("Error downloading Page Metadata:" + e.getMessage());
+        }
+    }
```

Comment:
The `PageMetadata` created by Classic is a `.xls` file with multiple tabs of data.

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/05ff05a3-d1f0-4a6e-865e-d4286df0249a)


The `.csv` produced by this endpoint is `108 KB` whereas `PageMetadata.xls` is 1.5 MB

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/6a926873-b52e-4a0d-8835-4084e812c7cb)